vizro-ai/docs/pages/user-guides/create-complex-dashboard.md

```diff
@@ -102,10 +98,10 @@ Next, submit the data and prompt string:
 from vizro_ai import VizroAI
 
 vizro_ai = VizroAI(model="gpt-4o")
-dashboard = vizro_ai._dashboard([df1, df2], user_question)
+dashboard = vizro_ai.dashboard([df1, df2], user_question)
 ```
 
-The call to `_dashboard()` triggers the dashboard building process. Once Vizro-AI finishes the dashboard generation process, you can launch the dashboard with `build()`.
+The call to `dashboard()` triggers the dashboard building process. Once Vizro-AI finishes the dashboard generation process, you can launch the dashboard with `build()`.
```

Comment:
```suggestion
The call to `dashboard()` triggers the dashboard generation process. Once it is complete, you can launch the dashboard with `build()`.
```

Reply:
I'm sorry, I know I asked about this before, but I found it really hard to get my head around the discrepancy of language. Is my edit accurate? If so, please apply it wherever it crops up.

Reply:
Maybe we could do a combination. I like the word building, because that is also the name of the method you use (`.build`).

```suggestion
The call to `dashboard()` triggers the dashboard building process. Once Vizro-AI finishes this process, you can launch the dashboard with `build()`.
```